commons/src/main/java/org/infinispan/commons/util/concurrent/jdk8backported/BoundedEquivalentConcurrentHashMapV8.java

```diff
@@ -1413,7 +1434,7 @@ public void onEntryRemove(Node<K, V> e) {
                      SizeAndEvicting sizeEvict = currentSize.get();
                      // If the size was changed behind our back by a remove we
                      // need to detect that
-                     if (sizeEvict.size - sizeEvict.evicting < maximumSize) {
+                     if (sizeEvict.size - sizeEvict.evicting <= maximumSize) {
```

Comment:
Hrmm I am not sure we want this change.  This would make it so we would not evict a value when we have an exact size match.

Scenario:

maximumSize = 100

sizeEvict.size = 110
sizeEvict.evicting = 10

In this case they would equal as they should because we want to evict 10 entries.  In this case though we would only evict 9 since we decrement that value.